tests/test_modeling_tf_common.py

```diff
@@ -164,6 +164,46 @@ def test_forward_signature(self):
                 expected_arg_names = ["input_ids"]
                 self.assertListEqual(arg_names[:1], expected_arg_names)
 
+    def test_saved_model_creation(self):
```

Comment:
Isn't that tested already that model can be saved and reloaded in TF (the normal way)

Reply:
Ah ok, I see...I think we should maybe switch the default of `saved_model` to `False` to prevent backwards breaking changes.